downloadDirectory/src/test/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/BiologicalProcessAnnotationBuilderTest.java

```diff
@@ -81,14 +82,20 @@ public void biologicalProcessAnnotationBuilderWithCatalystTest() throws Exceptio
         Mockito.when(mockCrossReferenceInst.getAttributeValue(ReactomeJavaConstants.identifier)).thenReturn("1234");
         Mockito.when(mockReactionInst.getReferers(ReactomeJavaConstants.hasEvent)).thenReturn(mockEventReferralSet);
         Mockito.when(mockEventReferralInst.getAttributeValuesList(ReactomeJavaConstants.goBiologicalProcess)).thenReturn(mockGOBioProcessSet);
-        Mockito.when(mockGOBioProcessInst.getAttributeValue(ReactomeJavaConstants.accession)).thenReturn("123456");
+        Mockito.when(mockGOBioProcessInst.getAttributeValue(ReactomeJavaConstants.accession)).thenReturn("1234");
         Mockito.when(mockEventReferralInst.getAttributeValue(ReactomeJavaConstants.stableIdentifier)).thenReturn(mockStableIdentifierInst);
-        Mockito.when(mockStableIdentifierInst.getAttributeValue(ReactomeJavaConstants.identifier)).thenReturn("A12345");
-        BiologicalProcessAnnotationBuilder.processBiologicalFunctions(mockReactionInst);
+        Mockito.when(mockStableIdentifierInst.getAttributeValue(ReactomeJavaConstants.identifier)).thenReturn("A1234");
+        Mockito.when(mockReferenceEntityInst.getAttributeValue(ReactomeJavaConstants.identifier)).thenReturn("R1234");
+        Mockito.when(GOAGeneratorUtilities.getStableIdentifierIdentifier(mockEventReferralInst)).thenReturn("1234");
+        Mockito.when(GOAGeneratorUtilities.getSecondaryIdentifier(mockReferenceEntityInst)).thenReturn("R5678");
+        Mockito.when(GOAGeneratorUtilities.generateGOALine(mockReferenceEntityInst, GOAGeneratorConstants.BIOLOGICAL_PROCESS_LETTER, "GO:1234", "REACTOME:1234", GOAGeneratorConstants.TRACEABLE_AUTHOR_STATEMENT_CODE, "1234")).thenCallRealMethod();
+        List<String> goaLines = BiologicalProcessAnnotationBuilder.processBiologicalFunctions(mockReactionInst);
```

Comment:
A little nit-picky, but I like to separate the mocking and other setup logic from the method call using a new line.  The same between the method call and the assert statements afterwards.  That way it's clear you're using the "arrange", "act", "assert" pattern for laying out your tests.

Reply:
I can understand that, I made that change. Everyone loves white space for readability